File: BOJ/10826/A.py
import sys


sys.stdin = open("D:\Projects\AA\BOJ\\10826\input.txt")
input = sys.stdin.readline

N = int(input().rstrip())

arr = [0 for _ in range(10001)]

arr[1] = 1
arr[2] = 1
for i in range(3, N+1):
    arr[i] = arr[i-1] + arr[i-2]

print(arr[N])

File: BOJ/10826/A.cpp
#include <iostream>
#include <vector>
using namespace std;

vector<long long> arr(10001, 0);
int main()
{
    cin.tie(0);
    ios::sync_with_stdio(0);
    freopen("input.txt", "rt", stdin);
    int N;
    cin >> N;
    arr[1] = 1;
    arr[2] = 1;
    for (int i = 3; i <= N; i++)
    {
        arr[i] = arr[i - 1] + arr[i - 2];
    }
    cout << arr[N];
}